src/hooks/useTasks.js

```diff
@@ -31,15 +33,22 @@ function useOpenTasks(tasks, now, courtSettings) {
         .join('')
     : null
 
+  const incompleteTasks = useMemo(() => {
+    if (!convertedTasks) {
+      return null
+    }
+    return convertedTasks.filter(
+      task =>
+        !voidedDisputes.has(task.dispute.id) &&
+        task.phase !== DisputesTypes.Phase.Ended
+    )
+  }, [convertedTasks]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Was thinking that it would be better to make `convertedTasksPhasesKey ` the dependency here so we don't filter every second

Reply:
yeah, that was something that we were discussing yesterday night with @sohkai

Reply:
cc @sohkai

Reply:
Coming back a bit late on this, but yes, after discussion it sounds like we should've been using `convertedTasksPhasesKey` as @fabriziovigevani suggested.

Reply:
hahah yes, will create later a pr for updating this